queries.sql

```diff
@@ -10,3 +10,37 @@ SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE NOT name='Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+/* p2 */
+/* ROLEBACK UPDATE */
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] please add a query to check in this transaction to make sure that the `species` was updated.